e2e/support/helpers/e2e-ad-hoc-question-helpers.js

```diff
@@ -110,18 +110,35 @@ export function openTable({
   );
 }
 
+/**
+ *
+ * @typedef {{mode?: "notebook", limit?: number, callback?: function }} OpenTablesProps
+ */
```

Comment:
It'd be ideal if the query was formatted in exactly the same way that it gets formatted in the native query editor:


https://github.com/metabase/metabase/assets/6830683/1fd7717c-c864-4e01-964b-df84bae83d4c